internal/pkg/bulk/bulk_integration_test.go

```diff
@@ -70,12 +69,8 @@ func TestBulkCreate(t *testing.T) {
 			Name:  "Invalid utf-8",
 			Index: string([]byte{0xfe, 0xfe, 0xff, 0xff}),
 			Err: es.ErrElastic{
-				Status: 500,
-				Type:   "json_parse_exception",
-			},
-			AltErr: es.ErrElastic{
 				Status: 400,
-				Type:   "parse_exception",
```

Comment:
This is the correct error type, not `json_parse_exception`

Reply:
Thanks @michel-laterman! ❤️